src/forms/password/Password.tsx

```diff
@@ -110,8 +110,8 @@ export class Password extends React.PureComponent<PasswordProps, PasswordState>
                     <div className={utils.classNames(classNames)} style={{width: "100%"}}>
                         <div className="clr-input-wrapper">
                             <div
-                                className={utils.classNames(["clr-input-group", "clr-row", className])}
-                                style={{...style, paddingRight: "1rem"}}
+                                className={utils.classNames(["clr-input-group", className])}
+                                // style={{...style, paddingRight: "1rem"}}
```

Comment:
Why comment and not remove ?  lets not keep commented code

Reply:
done